bundle/manifests/system-wicd-nodes_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/name: windows-machine-config-operator
+    app.kubernetes.io/part-of: wicd
+  name: system-wicd-nodes
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - list
+- apiGroups:
+  - ""
+  resources:
+  - nodes
```

Comment:
looks like `nodes` resource was defined in L#21 and extended in L#27 that grants additional verbs (get, patch, update, watch) on both nodes and nodes/status.